bin/bundle

```diff
@@ -8,55 +8,63 @@
 # this file is here to facilitate running it.
```

Comment:
what about exclude rubocop for this file?

Reply:
Already excluded